### ISSUE TITLE
Add a progress bar for -u update

### ISSUE THREAD

**Issue by fdservices** [falconindy/pkgfile#41]
Would it be possible to add progress bars for the update procedure?

**Comment by falconindy** [comment]
Sure, anything is possible. It's tedious because transfers are already done in parallel. I won't be implementing this myself. Patches welcome following a concrete proposal about how exactly this would look/behave.
